### DISCUSSION TITLE
3.7 release?

### DISCUSSION THREAD

**Discussion by drewgilliam** [colmap/colmap#1349] [Q&A] [answered]
The last colmap release was 3.6 in June 2020 (over a year ago), are there any plans to actually make a 3.7 release?  I know there are some fixes after 3.6 that are needed for newer Visual Studio compilers, for example 9f3a75a.  Thanks!

**Comment by alexmyczko** [comment, marked as answer]
https://github.com/colmap/colmap/releases/tag/3.7 it's been out for a while now
